### DISCUSSION TITLE
Ubuntu software store has two versions

### DISCUSSION THREAD

**Discussion by michaelwooley** [flameshot-org/flameshot#2998] [General]
This perhaps isn't a bug (or, at least, is known) but I noticed something when I installed flameshot via the "ubuntu software" app.

In short:

- A search for "flameshot" yields 2 results that seem to be identical at first blush.
- They're not, though:
    - The first result is an old (but official, I think) version (v0.6.0 from `ubuntu-focal-universe (deb)`): 
        - **I experience significant issues with the old build (flashing screen).**
        - There is also a "potentially unsafe" warning attached to this result.
    - The second result is the newer version (from snap store): **This works perfectly, has the "safe" badge, etc.**
- _Both results share the same reviews._ There are a bunch of recent, negative reviews. Maybe they're legit issues. Or maybe the person just installed the wrong version?

Thanks for this tool. Happy to provide any info/debug help if you like.

---

![Two results](https://user-images.githubusercontent.com/8423171/204683763-72fe3af6-b62e-4012-8154-2d243fa2d272.png)
_The installed version is the working version._

---

![image](https://user-images.githubusercontent.com/8423171/204684212-9245c4b9-3cd1-4f49-bf94-6fff7d260703.png)
_Info in the older download._

---

```bash
❯ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.5 LTS
Release:        20.04
Codename:       focal
```

**Comment by mmahmoudian** [comment]
Thanks for the post, but:
1. We do not develop Ubuntu or apt
2. We do not maintain Ubuntu's distro packages
3. Flameshot V0.6.0 is already about 5 years old

Ubuntu has unfortunately started integrating snaps in software center results without proper visual clues. I think this is something you should communicate with Ubuntu people as a user experience issue.
